spaceship/lib/spaceship/connect_api/testflight/testflight.rb

```diff
@@ -204,13 +204,19 @@ def delete_beta_groups_from_build(build_id: nil, beta_group_ids: [])
           test_flight_request_client.delete("builds/#{build_id}/relationships/betaGroups", nil, body)
         end
 
-        def create_beta_group(app_id: nil, group_name: nil, is_internal_group: false, public_link_enabled: false, public_link_limit: 10_000, public_link_limit_enabled: false)
+        def create_beta_group(app_id: nil, group_name: nil, is_internal_group: false, public_link_enabled: false, public_link_limit: 10_000, public_link_limit_enabled: false, has_access_to_all_builds: nil)
+          if is_internal_group
+            has_access_to_all_builds = true if has_access_to_all_builds.nil?
+          else
+            # Access to all builds is only for internal groups
+            has_access_to_all_builds = nil
```

Comment:
I checked but creating a few beta groups on App Store Connect, and if `isInternalGroup` is `false`, the value of `hasAccessToAllBuilds` is ignored and in the response `hasAccessToAllBuilds` is set to nil. Of course I did also check that passing nil was fine on the real server.

Reply:
For a future PR, it might be nice to catch this and print a warning instead of silently ignoring the invalid parameter

Reply:
I'll make a new 1-line PR